surch/organization.py

```diff
@@ -19,199 +19,200 @@
 
 import requests
 
-from . import logger, repo, utils, constants
-
-REPO_DETAILS_API_URL = \
-    'https://api.github.com/{0}/{1}/repos?type={2}&per_page={3}&page={4}'
-ORG_DETAILS_API_URL = 'https://api.github.com/{0}/{1}'
-
-
-lgr = logger.init()
+from . import repo, utils, constants
 
 
 class Organization(object):
     def __init__(
             self,
             search_list,
             organization,
-            git_user,
-            git_password,
+            verbose=False,
+            git_user=None,
```

Comment:
try to combine variables according to their context - git_user and git_password should be declared one after the other